kinesis-test/src/main/java/com/hazelcast/jet/tests/kinesis/KinesisVerificationP.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.kinesis;
+
+import com.hazelcast.jet.core.AbstractProcessor;
+import com.hazelcast.jet.core.ProcessorSupplier;
+import com.hazelcast.jet.datamodel.Tuple2;
+import com.hazelcast.jet.impl.pipeline.SinkImpl;
+import com.hazelcast.jet.pipeline.Sink;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.map.IMap;
+
+import java.util.PriorityQueue;
+
+import static com.hazelcast.jet.core.ProcessorMetaSupplier.forceTotalParallelismOne;
+import static com.hazelcast.jet.impl.pipeline.SinkImpl.Type.TOTAL_PARALLELISM_ONE;
+
+public class KinesisVerificationP extends AbstractProcessor {
+
+    public static final String CONSUMED_MESSAGES_MAP_NAME = "KinesisTest_latestCounters";
+
+    private static final int QUEUE_SIZE_LIMIT = 4_000;
```

Comment:
What `sleepMsBetweenItem` we will use? Currently `DEFAULT_SLEEP_MS_BETWEEN_ITEM = 1` which means that this queue limit is really small. However I guess we will use higher `sleepMsBetweenItem` than `1`, right?

Reply:
Yes, I was thinking 20ms, let me change the default to that one too.